### ISSUE TITLE
question about mask prompt

### ISSUE THREAD

**Issue by sove45** [MedicineToken/Medical-SAM2#59]
I noticed that you mentioned 'maskprompt' in the paper, but it was not reflected in the code. Could you explain how to use the mask from the first frame as a prompt to fine-tune the neural network
我注意到您在论文中有提到maskprompt，可是代码中没有体现，请问怎么使用第一帧的mask作为prompt来微调神经网络呢

**Comment by jiayuanz3** [comment]
for example, replace the click prompt with mask prompt in sam_prompt_encoder: https://github.com/MedicineToken/Medical-SAM2/blob/48975419fb65820fbd02dd86f46ce3a561a5a785/func_2d/function.py#L151
